routers/repo/editor_test.go

```diff
@@ -5,6 +5,8 @@
 package repo
 
 import (
+	"code.gitea.io/gitea/modules/git"
+	"code.gitea.io/gitea/modules/test"
```

Comment:
These should go below with "code.gitea.io/gitea/models" per normal styling preferences